core/model/modx/processors/browser/browser.class.php

```diff
@@ -76,6 +76,24 @@ public function getSource()
         return $this->source;
     }
 
+    /**
+     * Get the list of protected directories
+     *
+     * @return array
+     */
+    public function getProtectedPathDirectories() {
```

Comment:
Does it make sense to make it static as it returns constants? Not a request, but just a question for discussion.